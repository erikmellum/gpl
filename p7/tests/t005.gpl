on space {print("space");}
on leftarrow {print("leftarrow");}
on rightarrow {print("rightarrow");}
on uparrow {print("uparrow");}
on downarrow {print("downarrow");}
on f1 {print("f1");}
on akey {print("akey");}
on skey {print("skey");}
on dkey {print("dkey");}
on fkey {print("fkey");}
on hkey {print("hkey");}
on jkey {print("jkey");}
on kkey {print("kkey");}
on lkey {print("lkey");}
