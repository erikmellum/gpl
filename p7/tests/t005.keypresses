 





a
s
d
f
h
j
k
l
q
