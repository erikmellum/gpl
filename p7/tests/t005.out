gpl.cpp::main()
  input file(tests/t005.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  print_symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
gpl.cpp::main() Calling window->initialize().
gpl.cpp::main() Passing control to window->main_loop().
gpl[1]: space
gpl[2]: leftarrow
gpl[3]: rightarrow
gpl[4]: uparrow
gpl[5]: downarrow
gpl[6]: f1
gpl[7]: akey
gpl[8]: skey
gpl[9]: dkey
gpl[10]: fkey
gpl[11]: hkey
gpl[12]: jkey
gpl[13]: kkey
gpl[14]: lkey

Printing the symbol table from signal_handler()
